### DISCUSSION TITLE
Mega盘驱动，无法查看分享了的文件

### DISCUSSION THREAD

**Discussion by v3i1y** [AlistGo/alist#4447] [General]
加载了Mega网盘以后，AList里只显示了在mega里没有分享过的文件，所有分享了的文件在AList里都没有，这是feature还是bug？

**Comment by phytomorphogenesis** [comment]
最近发现了这个问题搜到了这条，早年创建分享的可以识别，新建的只要一分享就消失，分享链接或者享文件夹都会消失，尝试在可识别账号上再新建文件夹-alist可以识别，分享-刷新alist-消失，所以这也不是账号的问题。。。
